player/lua/osc.lua

```diff
@@ -70,7 +70,10 @@ local user_opts = {
     top_buttons_color = "#FFFFFF",	-- color of top buttons
     held_element_color = "#999999",	-- color of an element while held down
 
-    time_pos_outline_color = "#000000"	-- color of the border timecodes in slimbox and TimePosBar
+    time_pos_outline_color = "#000000",	-- color of the border timecodes in slimbox and TimePosBar
+
+    tick_delay = 0.03,                    -- minimum interval between OSC redraws in seconds
```

Comment:
Maybe merge both options? Use `-1` as special value? Ale might be useful to fallback to default tick duration if display_fps becomes unavailable during run-time.

`mp.observe_property('display-fps'` to set `display_fps`, `nil` if not available. Then could do 

``` lua
    local tick_delay = (user_opts.tick_delay ~= -1 and user_opts.tick_delay or display_fps) or default_tick_delay
```
Or similar, if you agree.

Reply:
For default value, I think we should at least make it 60 fps. I like your previous 0.01.

Reply:
> This should be a better value since high FPS displays are also trending to be multiple of 60 Hz.

I already mentioned it, but they are trending to be multiple of 24 fps I think, which makes more sense for movie playback. 
72 Hz
120 Hz
144 Hz
165 Hz
175 Hz
240 Hz
...
I think 144 Hz is still very common target, while there are in-between that are neither multiple of 24 nor 60.

I don't mind having it as 60 fps though, but either way if someone cares about smoothness, they should enable display fps option.

Reply:
Good talk, I guess.